.github/workflows/test.yml

```diff
@@ -79,7 +86,14 @@ jobs:
       working-directory: ${{runner.workspace}}/build
       run: |
         cmake $GITHUB_WORKSPACE -DCMAKE_BUILD_TYPE=${{ matrix.type }}
-      if: ${{ matrix.os == 'macos-12' }}
+      if: ${{ matrix.os == 'macos-12' && matrix.target-platform != 'iphoneos' && matrix.target-platform != 'iphonesimulator' }}
```

Comment:
why not check for target-platform == host-os?

Reply:
Changed.